website/docs/r/dns_domain_registration_nameservers.html.markdown.orig

```diff
@@ -0,0 +1,93 @@
+---

```

Comment:
This file has some conflicts <<HEAD....

Reply:
Junk file artifact of merging conflicts. Removed.